.github/workflows/node.js.yml

```diff
@@ -0,0 +1,28 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
+    - run: npm test
```

Comment:
the package.json doesn not contain these commands, why are they here :cold_sweat: 
```JSON
"scripts": {
    "test": "jest",
    "start:dev": "nodemon src/app.js"
  },
```

Reply:
Hi @YashKumarVerma, the command `npm ci` is the same as `npm install` but optimized for running in continuous integration environments and the command `npm test` is just a shortcut to `npm run test` that will execute the jest test suite on the application. 
But I've found a bug in the badge, it was markdown inside an html block. I'll update the PR with this fix.